backend/game/src/data/components/agenda/mod.rs

```diff
@@ -5,31 +5,44 @@ use crate::{data::common::expansions::Expansion, gameplay::player::PlayerId};
 
 use super::{objectives::secret::SecretObjective, planet::Planet, strategy_card::StrategyCard};
 
+/// All informatioin concerning an agenda.
```

Comment:
hmmm, is three clippy spellcheck lint? ;)

Reply:
schh x)